Doc/whatsnew/2.7.rst

```diff
@@ -2034,15 +2034,15 @@ version 1.3.  Some of the new features are:
   "xml" (the default), "html", or "text".  HTML mode will output empty
   elements as ``<empty></empty>`` instead of ``<empty/>``, and text
   mode will skip over elements and only output the text chunks.  If
-  you set the :attr:`tag` attribute of an element to ``None`` but
+  you set the :attr:`!tag` attribute of an element to ``None`` but
   leave its children in place, the element will be omitted when the
   tree is written out, so you don't need to do more extensive rearrangement
   to remove a single element.
 
   Namespace handling has also been improved.  All ``xmlns:<whatever>``
   declarations are now output on the root element, not scattered throughout
   the resulting XML.  You can set the default namespace for a tree
-  by setting the :attr:`default_namespace` attribute and can
+  by setting the :attr:`!default_namespace` attribute and can
```

Comment:
Hmmm... I think this might still exist (though I'm not an xml expert, by any means)... are we sure this one shouldn't be fixed by documenting it?

Reply:
Yep, let's revert for future consideration.
```suggestion
  by setting the :attr:`default_namespace` attribute and can
```